assets/scripts/menus/MenuBar.jsx

```diff
@@ -0,0 +1,170 @@
+    const signInVisibilityStyle = userId
+      ? { display: 'none' } : {}
+
+    // Buttons have `disabled={false}` because
+    // Firefox sometimes disables some buttons… unsure why
+    // --
+    // Note on translations - the {t()} method doesn't re-render when the
+    // language changes, so this still uses the `data-i18n` method to pick up
+    // on text content changes. Individual menus will re-render with {t()}.
+    return (
+      <nav className='menu-bar'>
+        <ul className='menu-bar-left'>
+          <li className='menu-bar-title'>
+            <div className='streetmix-logo' />
+            <h1>Streetmix</h1>
+          </li>
+          <li id='help-menu-item'>
+            <button
+              data-name='help'
+              data-i18n='menu.item.help'
+              className='menu-attached'
+              disabled={false}
+              onClick={this.onClickMenuButton}
+            >
+              Help
+            </button>
+          </li>
+          <li id='contact-menu-item'>
+            <button
+              data-name='contact'
+              data-i18n='menu.item.contact'
+              className='menu-attached'
+              disabled={false}
+              onClick={this.onClickMenuButton}
+            >
+              Contact
+            </button>
+          </li>
+        </ul>
+        <ul className='menu-bar-right'>
+          <li id='identity-menu-item' style={identityMenuVisibilityStyle}>
+            <button
+              data-name='identity'
+              className='menu-attached'
+              disabled={false}
+              onClick={this.onClickMenuButton}
+            >
+              <div className='avatar' data-user-id={userId} />
+              <span className='user-id'>{userId}</span>
+            </button>
+          </li>
+          <li id='sign-in-menu-item' style={signInVisibilityStyle}>
+            <a href={`/${URL_SIGN_IN_REDIRECT}`} className='command' data-i18n='menu.item.sign-in'>
+              Sign in
+            </a>
+          </li>
+          <li>
+            <a href='/new' target='_blank' data-i18n='menu.item.new-street'>
+              New street
+            </a>
+          </li>
+          <li id='gallery-link'>
+            <a href={myStreetsLink} data-i18n='menu.item.my-streets' onClick={onMyStreetsClick}>
+              My streets
+            </a>
+          </li>
+          {
+            (() => {
+              if (debug.experimental) {
+                return (
+                  <li id='settings-menu-item'>
+                    <button
+                      data-name='settings'
+                      data-i18n='menu.item.settings'
+                      className='menu-attached'
+                      disabled={false}
+                      onClick={this.onClickMenuButton}
+                    >
+                      Settings
+                    </button>
+                  </li>
+                )
+              }
+            })()
```

Comment:
Thanks Lou! I’ll look into what you sent.

> On Sep 7, 2016, at 11:25 AM, Lou Huang <notifications@github.com> wrote:
> 
> In assets/scripts/menus/MenuBar.jsx <https://github.com/codeforamerica/streetmix/pull/582#discussion_r77845132>:
> 
> > +              if (debug.experimental) {
> > +                return (
> > +                  <li id='settings-menu-item'>
> > +                    <button
> > +                      data-name='settings'
> > +                      data-i18n='menu.item.settings'
> > +                      className='menu-attached'
> > +                      disabled={false}
> > +                      onClick={this.onClickMenuButton}
> > +                    >
> > +                      Settings
> > +                    </button>
> > +                  </li>
> > +                )
> > +              }
> > +            })()
> @treyhahn <https://github.com/treyhahn> You can try out no-internet mode by appending ?debug-force-no-internet to the URL string like so: http://streetmix.net/-/421124?debug-force-no-internet. It can be turned on in a deployment by setting a NO_INTERNET_MODE environment variable to true. It's briefly mentioned here: https://github.com/codeforamerica/streetmix/blob/master/CONTRIBUTING.md#on-all-systems <https://github.com/codeforamerica/streetmix/blob/master/CONTRIBUTING.md#on-all-systems> (but not otherwise well documented).
> 
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub <https://github.com/codeforamerica/streetmix/pull/582/files/56682ae23c08450123b8fcb56450cf98d53b69ec#r77845132>, or mute the thread <https://github.com/notifications/unsubscribe-auth/AL-HbF-R4wOga2Ae0dAoQuC-OFyg_mdgks5qntd0gaJpZM4J1Zkn>.
>